spec/std/exception/call_stack_spec.cr

```diff
@@ -13,16 +13,31 @@ describe "Backtrace" do
     _, output, _ = compile_and_run_file(source_file)
 
     # resolved file line:column
-    output.should match(/#{source_file}:3:10 in 'callee1'/)
-
-    unless output =~ /#{source_file}:13:5 in 'callee3'/
-      fail "didn't find callee3 in the backtrace"
-    end
```

Comment:
Actually, these assertions were broken before this change, returning `true` even for absolute paths (since regex pattern used is not anchored)